### DISCUSSION TITLE
Is it posible to offload on Tenstorrent chips?

### DISCUSSION THREAD

**Discussion by stalkerg** [tinygrad/tinygrad#2146] [General]
I am just a little bit worried because the Tenstorrent chip supports reflecting node graphs in hardware, but because Tinygrad supports only optimization per operation basis, I don't know how it is possible to use Buda graph optimizer. 
I think this is a good explanation for such an architecture:
https://www.youtube.com/watch?v=lPX1H3jW8ZQ

**Comment by geohot** [comment]
Are tenstorrent chips real? Where can I rent/buy them?

**Reply by Codys12** [reply]
Very real now, any update on this?
